apps/client/components/Common/Modal/AlertModal/AlertModal.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, Story } from "@storybook/react";
+import type { ComponentProps } from "react";
+import AlertModal from ".";
+
+type Props = ComponentProps<typeof AlertModal>;
+
+const Template: Story<Props> = args => <AlertModal {...args} />;
```

Comment:
```suggestion
const Template: ComponentStoryFn<typeof AlertModal> = args => (
  <AlertModal {...args} />
);
```

이렇게 한 줄로 줄일 수 있습니다! 제가 스토리북 지식이 부족해서 혹시 이것이 문제가 된다면 답글 부탁드립니다!

Reply:
네네 이렇게 줄일 수 있지만, 각 단계를 명확하게 하는것도 중요하다고 생각하고
Props라는 타입을 반복해서 사용해야하는경우가 있어서 분리해뒀습니다!!

좋은 의견인것 같아서 감사합니다~~!

Reply:
넵 도움이 되었다면 히히

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ